src/index.md

```diff
@@ -10,10 +10,14 @@ As part of the [Code for America Brigade network](https://brigade.codeforamerica
 <div class="landing-page-image">
     <img alt="RSVP on Meetup" src="/assets/images/RSVP-on-meetup-300x200.png" />
 </div>
-<div>
+<p>
     We meet every Tuesday at 6:30pm at Oakland City Hall in Hearing Room 4. Food is provided. Join us by <a href="https://www.meetup.com/OpenOakland/">RSVPing on our Meetup page</a>.
-</div>
-
-<!-- AL removed images -->
+</p>
 
-OpenOakland members primarily use Slack for all communication. Please fill out [this contact form](https://tinyurl.com/y722n6ul) to get an invitation to our slack channel.
+<div class="landing-page-image">
+    <img alt="RSVP on Meetup" src="/assets/images/OO-on-Slack-300x128.png" />
+</div>
+<p>
+    OpenOakland members primarily use Slack for all communication. Please fill out <a href="https://tinyurl.com/y722n6ul">this contact form</a> to get an invitation to our slack channel.
+    OpenOakland members primarily use Slack for all communication. Please fill out [this contact form](https://tinyurl.com/y722n6ul) to get an invitation to our slack channel.
```

Comment:
Looks like this sentence was left over with the older markdown syntax after you replaced it with html!